CHANGES

```diff
@@ -9,6 +9,29 @@
 
  Changes between 1.1.1 and 3.0.0 [xx XXX xxxx]
 
+  *) Added the X509_LOOKUP_METHOD X509_LOOKUP_store, to allow access to
```

Comment:
Something about this sentence doesn't quite scan right

Reply:
How so?  "Added the <TYPE> <name-of-instance>."

Reply:
Ahh!! Doh! Actually if that phrasing wasn't immediately clear to me, then others will probably struggle. Perhaps "Added a new X509_LOOKUP_METHOD called X509_LOOKUP_store,..."

Reply:
Ok, I can do that